tutorials/tutorial4_certified_over_estimation.ipynb

```diff
@@ -47,16 +33,10 @@
     "hardware constraints, and real-time constraints. \n",
     "Also, we need to prove that the decisions made by the system using the surrogate model \n",
     "instead of the reference one will be **safe**. **The confidence in its safety has to be demonstrated \n",
-    "to certification authorities**. \n"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "id": "55ed195d",
-   "metadata": {},
-   "source": [
+    "to certification authorities**. \n",
+    "\n",
     "- When running this notebook on Colab, we need to install *decomon* if on Colab. \n",
-    "- If you run this notebook locally, do it inside the environment in which you [<span style='color:purple'>installed *decomon*</span>](https://airbus.github.io/decomon/main/install.html) as well as [scipy](https://scipy.org/)."
+    "- If you run this notebook locally, do it inside the environment in which you [<span style='color:purple'>installed *decomon*</span>](https://airbus.github.io/decomon/main/install.html) as well as [scipy](https://scipy.org/).\n"
```

Comment:
Do we want to add the installation colab stuff in the description that will be in the online doc ? 
Because putting it in the first cell will imply that.